.travis.yml

```diff
@@ -0,0 +1,8 @@
+lagunage: python
+python:
+  - 2.7
+before_script:
+  - git clone https://github.com/common-workflow-language/schema_salad.git salad && cd ./salad && pip install . --user && cd ..
+script:
+  - for target in v1.0/v1.0/*.cwl;     do schema-salad-tool ./salad/schema_salad/tests/test_schema/CommonWorkflowLanguage.yml ${target} || exit 1; done
+  - for target in v1.0/examples/*.cwl; do schema-salad-tool ./salad/schema_salad/tests/test_schema/CommonWorkflowLanguage.yml ${target} || exit 1; done
```

Comment:
Thanks for this commit https://github.com/common-workflow-language/common-workflow-language/commit/915d49f8fb912774cbf5cbe6ba910a430ef69941, `v1.0/examples/*.cwl` is no longer to be tested. I'll fix this pull-request.